src/Controller/CaseController.php

```diff
@@ -38,10 +42,60 @@ public function summary(CaseEntity $case): Response
     /**
      * @Route("/{id}/information", name="case_information", methods={"GET"})
      */
-    public function information(CaseEntity $case): Response
+    public function information(CaseEntity $case, ComplaintCategoryRepository $categoryRepository): Response
     {
+        /** @var ComplaintCategory $complaintCategory */
+        $complaintCategory = $categoryRepository->findOneBy([
+            'board' => $case->getBoard(),
+            'municipality' => $case->getMunicipality(),
+            'name' => $case->getCaseType(),
+        ]);
+
         return $this->render('case/information.html.twig', [
             'case' => $case,
+            'complaintCategory' => $complaintCategory,
+        ]);
+    }
+
+    /**
+     * @Route("/{id}/information/edit", name="case_information_edit", methods={"GET", "POST"})
+     */
+    public function editInformation(CaseEntity $case, ComplaintCategoryRepository $categoryRepository, Request $request): Response
+    {
+        /** @var ComplaintCategory $complaintCategory */
+        $complaintCategory = $categoryRepository->findOneBy([
+            'board' => $case->getBoard(),
+            'municipality' => $case->getMunicipality(),
+            'name' => $case->getCaseType(),
+        ]);
+
+        $complaintCategories = $categoryRepository->findBy([
+            'board' => $case->getBoard(),
+            'municipality' => $case->getMunicipality(),
+        ]);
+
+        // Todo: Handle other case types, possibly via switch on $case->getBoard()->getCaseFormType()
+        $form = $this->createForm(ResidentComplaintBoardCaseType::class, $case, ['board' => $case->getBoard()]);
+
+        $form->handleRequest($request);
+        if ($form->isSubmitted() && $form->isValid()) {
+            $case = $form->getData();
+
+            $em = $this->getDoctrine()->getManager();
+            $em->flush();
+
+            return $this->redirectToRoute('case_information', [
+                'id' => $case->getId(),
+                'case' => $case,
+                'complaintCategory' => $complaintCategory,
```

Comment:
This is probably not needed as complaint category should be a property on the case entity class.

Reply:
Removed as so longer necessary per addition of complaintCategory property on CaseEntity.